### PR TITLE
Add styling and any missing JSX elements to your project (without add…

### PR DESCRIPTION
**Pull request by MMGGYY66**
# Add styling and any missing JSX
elements to your project (without adding extra functionality)

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -2,7 +2,6 @@
 @import url('https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200');
 @import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
```
Comment (Olamarx) [inline src/index.css]: - [ ] I will implore you to style the app exactly as the [Zeplin](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378/stage) app is. As of now, the background color is different.

### Zeplin design
image
### Your design
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @MMGGYY66,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 2]
```diff
@@ -95,7 +92,7 @@ a {
   line-height: normal;
   font-weight: bold;
   letter-spacing: 1.9px;
-  color: #240b0b;
+  color: #1c1111;
 }
```
Comment (binhussen) [inline src/index.css:96]: - Great Job for modifying the style as the previous code reviewer suggested you :+1: I think you need to do a little bit to make the font, bg, and colors your, you can use [Zeplin](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378/stage) inspection :+1: 
here is your design
Screenshot from 2023-03-02 18-03-10
here is zipline design
Screenshot from 2023-03-02 18-04-13

**Review by binhussen — APPROVED** [review]
Hi @MMGGYY66 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- used react-redux ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by binhussen** [comment]
- @MMGGYY66 Kindly this PR was approved by mistake on your dashboard, please continue on the given suggestion it will be fixed soon as request review :+1:

**Comment by MMGGYY66** [comment]
> * @MMGGYY66 Kindly this PR was approved by mistake on your dashboard, please continue on the given suggestion it will be fixed soon as request review 👍

I fixed it. thanks.

**Comment by enniopo** [comment]
Hi @MMGGYY66, I updated the status of the _Bookstore: styling_ activity on your dashboard. 
When you are ready, you can submit another code review request. 
Thanks, and apologies for the inconvenience.

**Comment by binhussen** [comment]
Great job for updating it, but you have to request a new review after your dashboard has been updated

**Pull request merged**